static/js/message_list.js

```diff
@@ -673,6 +673,14 @@ $(document).on('message_selected.zulip zuliphashchange.zulip mousewheel', functi
     viewport.stop_auto_scrolling();
 });
 
+$(window).load(function () {
+    $('img.emoji').each(function () {
```

Comment:
@timabbott For this to take effect in all the images, "img.emoji" should be replaced by "img". 
I was thinking of putting a border around the images that are yet to load, and removing it as soon as it gets loaded. Is it the right approach?